### PR TITLE
Updated documentation to download engine from maven central

### PR DESCRIPTION
**Pull request by mcarmonaa**
- Updated README to point to maven central when use engine as a dependency
- Changed examples to use engine dependency from maven central

### CODE REVIEW

**Comment by bzz** [comment]
Same question as in https://github.com/src-d/engine/pull/138/files#r147057446

**Comment by mcarmonaa** [comment]
@bzz just use the last version on maven central, I'm updating https://github.com/src-d/engine/pull/138 to use version `0.1.2`

**Comment by mcarmonaa** [comment]
My fault, the last `siva-java` version on `maven-central` is `0.1.1`, I'll update this PR instead #138

**Comment by mcarmonaa** [comment]
Sorry about the confusion, versions are right:
`tech.sourced:engine:0.1.2`
`tech.sourced:siva-java:0.1.1`

**Comment by mcarmonaa** [comment]
@ajnavarro @bzz @erizocosmico , please review again the latest changes:

- References to specific engine version `0.1.2` have been changed by `[version]`.
- Instructions and links to [maven central](http://search.maven.org/#search%7Cga%7C1%7Ctech.sourced) have been added to replace `[version]` with the right version.

**Comment by erizocosmico** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -14,13 +14,15 @@ Current implementation combines:
...
+
 ```bash
 $ wget "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=spark/spark-2.2.0/spark-2.2.0-bin-hadoop2.7.tgz"
 $ tar -xzf spark-2.2.0-bin-hadoop2.7.tgz; cd spark-2.2.0-bin-hadoop2.7
-$ ./bin/spark-shell --packages "com.github.src-d:engine:master-SNAPSHOT" --repositories "https://jitpack.io"
+$ ./bin/spark-shell --packages "tech.sourced:engine:[version]"
 
 # or
-$ ./bin/pyspark --repositories "https://jitpack.io"  --packages "com.github.src-d:engine:master-SNAPSHOT"
+$ ./bin/pyspark --packages "tech.sourced:engine:0.1.2"
```
Comment (ajnavarro) [inline README.md]: [version]

**Comment by bzz** [comment]
Shall we merge this guy, or is there anything left to do?

**Comment by mcarmonaa** [comment]
@bzz, waiting for @ajnavarro to review the requested changes and approve them before merge it.

**Pull request merged**